pkg/combustion/templates/15-rke2-multi-node-installer.sh.tpl

```diff
@@ -53,6 +53,8 @@ cp {{ .registryMirrors }} /etc/rancher/rke2/registries.yaml
 export INSTALL_RKE2_TAR_PREFIX=/opt/rke2
 export INSTALL_RKE2_ARTIFACT_PATH={{ .installPath }}
 
+mkdir -p /opt/cni
```

Comment:
Can we add a comment describing that this is necessary for SELinux? Without context or any other usage it's not clear to other developers what we're doing here.

Reply:
Done!